### ISSUE TITLE
(bug?) only dark capped mushroom grows into a big shroom in mycelium

### ISSUE THREAD

**Issue by trollworkout** [ganymedes01/TheErebus#431]
seems i can't naturally grow any of the big shrooms 

except for dark capped mushroom which is the only one that grows into  a large mushroom 

in in a mushroom shore island biome and there is mycelium all over 

when i plant erebus shrooms only dark capped grows. but all shrooms spread and grow naturally  is just when you bonemeal them they don't grow big size.

**Comment by vadis365** [comment]
Some of the giant mushrooms require a 2x2 (like big trees)

**Comment by trollworkout** [comment]
@vadis365  i didn't know this i will try again thank you for the tip.

**Comment by trollworkout** [comment]
@vadis365  i tried 2 x 2, 3 x 3, 4 x 4 nothing works. unless they need darkness maybe they only grow underground . i am using mycelium. maybe they need something else. dirt?

they don't seem to grow anywhere on any material in any condition in any number

i must ASSUME is either i am missing something or it's likely they were never programmed to grow like dark capped mushroom does

**Comment by ganymedes01** [comment]
Some need a 2x2 (not 3x3 or 4x4) and all of them need a big enough area around and above them to grow.
Light levels aren't important, they'll grow regardless.

**Comment by trollworkout** [comment]
@ganymedes01  i have tried on mycelium 2 x 2 with lots of room around them and they won't grow :/

**Comment by ganymedes01** [comment]
maybe there wasn't enough room above them? Some of these mushrooms can get quite big, you might be underestimating their size.

Could you post a screenshot of what you're doing?

Do you have any mod that might be messing with air blocks?

**Comment by ganymedes01** [comment]
Also, is the mushroom you're trying to grow called sarcastic czech? Because those haven't been implemented yet

**Comment by trollworkout** [comment]
@ganymedes01 i have tried all the unique ones dark capped, gandma shoes, sarcastic czech, kaizer fingers and dutch cap. none of these grow on mycelium 4 x 4. let me try again right now. 

UNLESS! maybe the grow naturally and do not need bonmeal? i am trying to use bonemeal to grow them like the regular minecraft shrooms. only dark capped one works. 

after bonemeal
![screen shot 2015-11-24 at 3 18 39 pm](https://cloud.githubusercontent.com/assets/6987027/11379506/a77d0caa-92be-11e5-90a6-e397cf81536c.png)

**Comment by trollworkout** [comment]
or maybe they need like 1 hour 64 64 stacks of bonemeal to grow

**Comment by trollworkout** [comment]
@ganymedes01  i managed to make 1 grow..i figured it out. i think they need a really big base to grow on

**Comment by ganymedes01** [comment]
Which one?

**Comment by trollworkout** [comment]
@ganymedes01  dutch one i managed to make it grow..but i think that kaizer and czech are not implemented yet. i made a huge base with so much empty space and still not growing.

**Comment by trollworkout** [comment]
@ganymedes01  so dutch and grandma grow if the base it at least about 7 x 7 but the other ones even with a base of 50 x 50 still don't grow so i think they are not implemented

**Comment by ganymedes01** [comment]
Look, I can see the code, I know they are implemented =P All but the sarcastic czech.

**Comment by trollworkout** [comment]
@ganymedes01  hmm so dutch needs 6 x 6 area..grandma needs 4 x 4 

lemme try again on kaizer

**Comment by trollworkout** [comment]
@ganymedes01  yea i can't make the other one grow . i m gonna read trough this code see what requirements it has if is impemented is must be some insane requirements for it to grow if it doesn't wanna grow on a 50 x 50 area of mycelium right in the center

**Comment by trollworkout** [comment]
@ganymedes01  i guess im happy the other 2 can grow at least 3 / 5 are working .

**Comment by ganymedes01** [comment]
4/5 grow

**Comment by ganymedes01** [comment]
Before:
<img width="646" alt="screen shot 2015-11-24 at 20 42 29" src="https://cloud.githubusercontent.com/assets/5294529/11380093/f2b0385a-92eb-11e5-83ef-a284a521e7ed.png">

After:
<img width="496" alt="screen shot 2015-11-24 at 20 42 34" src="https://cloud.githubusercontent.com/assets/5294529/11380097/f6fc687a-92eb-11e5-9d17-67d544075697.png">

**Comment by ganymedes01** [comment]
Oh and here's the code showing that the mushroom IS IMPLEMENTED:
https://github.com/ganymedes01/TheErebus/blob/master/src/main/java/erebus/world/feature/plant/WorldGenGiantMushrooms.java#L111

**Comment by vadis365** [comment]
![2015-11-24_20 48 20](https://cloud.githubusercontent.com/assets/6518708/11380240/c5eb7f5e-92ec-11e5-9502-b2ba20d9c6ef.png)
All works fine for me.

Kaizer's Fingers & Dark Capped are singles, Dutchcap and Grandma's Shoes are 2x2.
